### PR TITLE
[DOC] Added Code Copying Functionality Across Repository Sections

### LINKED ISSUE #5183: [DOC] Add "Copy Code" Functionality to All API Reference Pages
**What do you want to do?**
 
- [x] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 
I have noticed that a lot of pages in the API reference section have inconsistencies in providing the option for users to copy the code snippets.

#### Screenshots:
In the "_Example_" section of [page](https://opensearch.org/docs/latest/api-reference/index-apis/close-index/), a helpful feature is provided which allows users to easily copy the JSON code.
image

However, in the "Path and HTTP methods" section, this option is missing.
image

This inconsistency could potentially confuse users and make it less convenient for them to use the provided code. To enhance the user experience and maintain consistency across all pages of the API reference, I would like to include the copy option for all relevant code snippets throughout the document.

### PR DESCRIPTION
**Pull request by KiranAminPanjwani**
### Description
I've addressed various pages throughout the entire repository, including the API reference, Troubleshooting, Tools, and Data Prepper sections, to rectify inconsistencies in code snippet copying options. The following measures have been implemented:

- Verified that users can seamlessly copy commands, request bodies, examples, and request body formats.
- Implemented one-click copy functionality in previously missing areas to save users time and enhance their overall experience.

### Issues Resolved
Closes #5183 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by KiranAminPanjwani** [comment]
@Naarcha-AWS Kindly review and merge my PR.😇

File: _api-reference/script-apis/exec-stored-script.md
Comment (Naarcha-AWS) [inline _api-reference/script-apis/exec-stored-script.md:257]: Why are we adding an additional tick mark here and other places?

**Review by Naarcha-AWS — APPROVED** [review]
A quick clarifying question but otherwise LGTM.

**Comment by kolchfa-aws** [comment]
@KiranAminPanjwani: The rendering that you see happens not because it's 4 back tics instead of 3, but because the number of back tics opening the code block must be the same as the number of back tics closing the code block. On this page, some requests/responses have 4 opening back tics but only 3 closing ones, which makes Markdown confused :) 

4 back tics are necessary to escape a code block which itself contains 3 back tics (i.e., to display 3 back tics you need to escape them with 4 back tics). In all our requests and responses, 3 back tics is enough.

**Comment by KiranAminPanjwani** [comment]
> > > @KiranAminPanjwani: The rendering that you see happens not because it's 4 back tics instead of 3, but because the number of back tics opening the code block must be the same as the number of back tics closing the code block. On this page, some requests/responses have 4 opening back tics but only 3 closing ones, which makes Markdown confused :)
> > > 4 back tics are necessary to escape a code block which itself contains 3 back tics (i.e., to display 3 back tics you need to escape them with 4 back tics). In all our requests and responses, 3 back tics is enough.
> > 
> > 
> > Acknowledged sir.✨ Should I edit the files again?
> 
> Yes.

Okay sir.

**Comment by KiranAminPanjwani** [comment]
@Naarcha-AWS, @kolchfa-aws I have updated the file again. Kindly review and please let me know if any further modifications are needed.

**Comment by Naarcha-AWS** [comment]
@KiranAminPanjwani: Should be good to merge once we fix the merge conflicts. Let me know if you need help with that.

**Comment by KiranAminPanjwani** [comment]
> @KiranAminPanjwani: Should be good to merge once we fix the merge conflicts. Let me know if you need help with that.

Sir, I would greatly appreciate your assistance, as I'm having trouble identifying the specific conflicts.

image

**Comment by KiranAminPanjwani** [comment]
> > @KiranAminPanjwani: Should be good to merge once we fix the merge conflicts. Let me know if you need help with that.
> 
> @Naarcha-AWS Sir, I would greatly appreciate your assistance, as I'm having trouble identifying the specific conflicts.
> 
> image

@Naarcha-AWS @kolchfa-aws Sir, Kindly guide me.✨

**Comment by Naarcha-AWS** [comment]
@KiranAminPanjwani: This should help you get started: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line.

If you need someone to walk you through it, feel free to send me a message and I would be happy to assist.

**Comment by Naarcha-AWS** [comment]
@KiranAminPanjwani: Do you still need help resolving these merge conflicts?

**Comment by KiranAminPanjwani** [comment]
> @KiranAminPanjwani: Do you still need help resolving these merge conflicts?

Sir, I'm working on it.

**Comment by KiranAminPanjwani** [comment]
> > @KiranAminPanjwani: Do you still need help resolving these merge conflicts?
> 
> Sir, I'm working on it.

@Naarcha-AWS Sir, I'm facing issues in resolving the conflicts at my end. Shall I create a new PR for this one?

**Comment by KiranAminPanjwani** [comment]
> > > @KiranAminPanjwani: Do you still need help resolving these merge conflicts?
> > 
> > 
> > Sir, I'm working on it.
> 
> @Naarcha-AWS Sir, I'm facing issues in resolving the conflicts at my end. Shall I create a new PR for this one?
Could we please connect on Discord to discuss my issue?

**Comment by kolchfa-aws** [comment]
@KiranAminPanjwani The "Path and HTTP methods" section does not need copy buttons. The purpose of this section is just to showcase the path and HTTP methods. For many requests, copying and running these would give an error because there is no payload for the request. Please remove copy buttons from these sections. Thanks!

**Comment by KiranAminPanjwani** [comment]
> @KiranAminPanjwani The "Path and HTTP methods" section does not need copy buttons. The purpose of this section is just to showcase the path and HTTP methods. For many requests, copying and running these would give an error because there is no payload for the request. Please remove copy buttons from these sections. Thanks!

Okay, sir. Should I remove the copy button only from where the conflicts occur, or should I remove my addition from all the files?

**Comment by kolchfa-aws** [comment]
Please remove the copy buttons from all "Path and HTTP methods" code snippets. Thank you!

**Comment by hdhalter** [comment]
@KiranAminPanjwani , Hi Kiran, are you planning on finishing up this PR? Thanks!

**Review by kolchfa-aws — CHANGES_REQUESTED** [review]
Just so this doesn't get accidentally merged, let's make sure the Path and HTTP methods section does not have copy buttons. Thanks!

**Comment by KiranAminPanjwani** [comment]
@Naarcha-AWS @hdhalter @kolchfa-aws 
Hi,
Yes, I'm working on it. I believe closing this pull request would be better, as it currently has conflicts. Therefore, I am closing this PR and will create a new one with the proper working.
Thank you.

**Pull request closed without merging**
